lusidtools/cocoon/cocoon.py

```diff
@@ -163,6 +163,15 @@ def load_holding_batch(
                 """There is no mapping for effective_at in the required mapping, please add it"""
             )
 
+        # If only an adjustment has been specified
+        if "holding_adjustments_only" in list(kwargs.keys()) and kwargs["holding_adjustments_only"]:
```

Comment:
Since holdings_adjustment_only is set default to False in load_from_data_frame(), wont `"holding_adjustments_only" in list(kwargs.keys())` always be true, making this condition unnecessary?

Reply:
Yes, this is true. Better safe than sorry though I guess.

Reply:
👍